src/components/content/Questions.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <div>
+    <div v-for="question in questions" :key="question.id">
+      <div v-html="question.body" />
+      <hr>
+    </div>
+  </div>
+</template>
+
+<script>
+  import {mapActions, mapGetters} from 'vuex'
+
+  export default {
```

Comment:
all inside this block should be on another level. In your case `computed` is on the same level with `export`, but should not be.